docs/readthedocs/source/doc/Friesian/QuickStart/Train_wnd.md

```diff
@@ -0,0 +1,163 @@
+Know more about W&D and it's implementation [here](https://github.com/intel-analytics/BigDL/blob/main/python/friesian/example/wnd/train/wnd_train_recsys.py)
+
+- An [**Orca Estimator**](../../Orca/Overview/distributed-training-inference.md) provides sklearn-style Estimator APIs to perform distributed [TensorFlow](https://github.com/tensorflow/tensorflow), [PyTorch](https://github.com/pytorch/pytorch), [Keras](https://github.com/keras-team/keras) and [BigDL](https://github.com/intel-analytics/BigDL) training and inference.
+
+## **2. W&D implemendation on Friesian**
+See the full demo code [here](https://github.com/intel-analytics/BigDL/tree/main/python/friesian/democode/train_wnd.py)
+
+### **2.1. Initialize OrcaContext**
+```python
+from bigdl.orca import init_orca_context
+sc = init_orca_context("local",  cores=8, memory="8g", init_ray_on_spark=True)
+```
+
+### **2.2. Load data into FeatureTable**
+Data can be loaded into Friesian FeatureTables from spark dataframe, pandas dataframe, parquet file, jason files, csv files and text files.
+```python
+from bigdl.friesian.feature import FeatureTable
+from bigdl.dllib.feature.dataset import movielens
+import pandas as pd
+
+data_dir = "./movielens"
+ratings = movielens.get_id_ratings(data_dir)
+ratings = pd.DataFrame(ratings, columns=["user", "item", "rate"])      
+ratings_tbl = FeatureTable.from_pandas(ratings)
+ratings_tbl.cache()
+
+user_tbl = FeatureTable.read_csv(data_dir + "/ml-1m/users.dat", delimiter=":")\
+    .select("_c0", "_c2", "_c4", "_c6", "_c8")\
+    .rename({"_c0": "user", "_c2": "gender", "_c4": "age", "_c6": "occupation", "_c8": "zip"})\
+    .cast(["user"], "int")
+
+user_tbl.cache()
+ratings_tbl.show(3, False)
+user_tbl.show(3, False)
+# 
+# +----+----+----+
+# |user|item|rate|
+# +----+----+----+
+# |1   |1193|5   |
+# |1   |661 |3   |
+# |1   |914 |3   |
+# +----+----+----+
+#
+# +----+------+---+----------+-------+
+# |user|gender|age|occupation|zipcode|
+# +----+------+---+----------+-------+
+# |1   |F     |1  |10        |48067  |
+# |2   |M     |56 |16        |70072  |
+# |3   |M     |25 |15        |55117  |
+# +----+------+---+----------+-------+
+```
+
+### **2.3. Data processing using FeatureTable**
+#### 2.3.1. Deal with missing data
+For missing values, one can replace them with a specified value or just simply drop the records with null values. For numerical columns, `feature_tbl.fill_median(columns)` can fill missing values with medians.
+```python
+user_tbl = user_tbl.fillna('0', "zipcode")
+```
+#### 2.3.2. Scale numerical features
+Generate continuous features like user stats and rescale these features within a range of min and max, one can also use `feature_tbl.transform_min_max_scale(min_max_dict)` to apply the scale of `min_max_dict` to data while preparing new feature table to make predictions.
```

Comment:
`feature_tbl.min_max_scale()`?

Reply:
feature_tbl.min_max_scale() takes an argument of dictionary

Reply:
The description does not match the code below

Reply:
revised